site/components/payment-streams/EditStream.js

```diff
@@ -217,12 +219,12 @@ const EditFundingAddress = function ({ stream }) {
         })
         // eslint-disable-next-line promise/catch-or-return
         updateStreamInfo({
+          account,
           id: streamId,
           lib: paymentStreamsLib,
-          account,
           streamsView: 'outgoing'
         }).then(() => mutate())
-        router.push('/payment-streams')
+        router.push(`${locale}/payment-streams`)
```

Comment:
Can we use the `useRouter` exported from `site/navigation.js` so we avoid manually adding the `locale` to the route? Does it affect anything else doing that change?

Reply:
The same applies to all the other places where `router` is changing route and manually pretending the `locale`

Reply:
right, I can use `useRouter` from `site/navigation.js` but it seems I still need nextjs's `useRouter` for the cases I need to read queryParams as the one from `navigation.js` doesn't return the `query` object.
Here: https://github.com/hemilabs/pure.finance/pull/1/commits/abcda61f8357cd38f6d8a77b5ddadf6a64b163b9